mypy_django_plugin/main.py

```diff
@@ -52,25 +52,28 @@ def add_new_manager_base(ctx: ClassDefContext) -> None:
 
 
 def extract_django_settings_module(config_file_path: Optional[str]) -> str:
-    errors = Errors()
+    usage = "\n".join(["(config)",
+                       "[mypy.plugins.django_stubs]",
+                       "\t(required) django_settings_module: str"])
+
+    def raise_error(handler: ErrorHandler, error_type: int) -> NoReturn:
+        messages = {1: "'django_settings_module' is not set: no mypy config file specified",
```

Comment:
Let's make an `enum` from keys here. Like `@unique class ErrorType(enum.IntEnum)`?

Reply:
Very good idea! I think enums access time is faster than dicts. 

Considering that they shouldn't get evaluated until needed for an error, where should they live?

Reply:
Just had a look into this and I have mixed feelings. Enums can take the following form:
```
class ErrorType(enum.Enum):
    SOME_ERROR: "some error because bad bad something"
    BAD_ERROR: "bad error because something"
# or, say
class ErrorType(enum.IntEnum):
    SOME_ERROR: 1
    BAD_ERROR: 2
```

As far as I understand, the `IntEnum` just allows the enum to be referenced like `ErrorType(1)`. The reason I used an int as the key initially was that it's the smallest type that would allow to reference errors without having to add those definitions to the scope before needed.

If an enum is used, this would require a call like `error(ErrorType.MISSING_SECTION`) or similar, i.e. they'd be in the scope before they are needed, which isn't great considering the startup time which is already rather slow and that errors won't get thrown often. 

Also, currently these messages aren't used anywhere else or have any operations done to them. If we, say, had to do any comparison operations, then attaching types to them with enums would make sense; but now they're rather limited, so I'd probably leave it as it is for now, mostly due to performance reasons though. 

Let me know if I'm missing out on something or you have something planned where having them as enums would make more sense. For now I've moved the error handler import to the local scope having done some `python -X importtime` runs - the startup is at least double as sluggish as mypy's :snail:

Reply:
Also added a try/except for the file read, and using `ConfigParser.read_file` instead of `read`, since the latter is made for multiple files and does some extras that aren't required here.